### PR TITLE
HParams: Switch runs data table to using new selector

### PR DESCRIPTION
**Pull request by rileyajones**
## Motivation for features / changes
A while back I introduced a series of new selectors needed to translate hparams data into columns while accounting for filtering and selection state. Apparently that never actually got plugged into the runs data table.

### CODE REVIEW

File: tensorboard/webapp/metrics/views/main_view/common_selectors.ts [hunk 1]
```diff
@@ -26,7 +26,6 @@ import {
   getRouteKind,
   getRunsFromExperimentIds,
   getColumnHeadersForCard,
-  getCardMetadata,
```
Comment (rileyajones) [inline tensorboard/webapp/metrics/views/main_view/common_selectors.ts:29]: This isn't necessary but I'm trying to remove unused import while I find them. Now that I'm on duty these lint errors might be a good thing to try to fix.

File: tensorboard/webapp/runs/views/runs_table/types.ts [hunk 2]
```diff
@@ -41,6 +41,7 @@ export interface RunTableItem {
   experimentAlias: ExperimentAlias;
   experimentName: string;
   selected: boolean;
+  runColor: string;
```
Comment (rileyajones) [inline tensorboard/webapp/runs/views/runs_table/types.ts:44]: Apparently this has just been wrong for years.

File: tensorboard/webapp/runs/views/runs_table/runs_table_test.ts [hunk 3]
```diff
@@ -3240,38 +3237,19 @@ describe('runs_table', () => {
...
           id: 'book2',
           color: '#111',
           run: 'The Chamber Of Secrets',
+          experimentName: 'book',
           selected: false,
         },
       ]);
     });
 
-    it('passes selected value of false if run is not in selectionMap', () => {
```
Comment (rileyajones) [inline tensorboard/webapp/runs/views/runs_table/runs_table_test.ts:3254]: This is no longer necessary, as the behavior is tested thoroughly in the underlying selector.

File: tensorboard/webapp/runs/views/runs_table/types.ts (hunk 2 above)
Comment (JamesHollyer) [reply tensorboard/webapp/runs/views/runs_table/types.ts:44]: I honestly do not understand how typescript did not catch this.

**Review by JamesHollyer — APPROVED** [review]

**Pull request merged**